paasta_tools/cleanup_marathon_jobs.py

```diff
@@ -63,14 +66,23 @@ def delete_app(app_id, client):
     happened"""
     log.warn("%s appears to be old; attempting to delete" % app_id)
     service, instance, _, __ = marathon_tools.deformat_job_id(app_id)
+    cluster = load_system_paasta_config().get_cluster()
     try:
         with bounce_lib.bounce_lock_zookeeper(marathon_tools.compose_job_id(service, instance)):
             bounce_lib.delete_marathon_app(app_id, client)
+            send_event(
+                service=service,
+                namespace=instance,
+                cluster=cluster,
+                soa_dir=DEFAULT_SOA_DIR,
```

Comment:
Er, we are using the default SOA_DIR when this is passed in via ARGS, can you pass it through from main?